### DISCUSSION TITLE
Unable to upload icon

### DISCUSSION THREAD

**Discussion by BlackWhite2000** [GitbookIO/community#392] [0. General bugs and questions]
![image](https://github.com/GitbookIO/community/assets/150109049/923e6b69-2093-42f7-bbee-4b7f98d2f7f9)
I can't upload this icon, I tried 132px, 200px png and ico formats
I can't get it to upload.
![image](https://github.com/GitbookIO/community/assets/39222378/a3c1d6da-70c2-41bb-a20c-b0397012561f)

![image](https://github.com/GitbookIO/community/assets/39222378/4a6bfd74-9db0-4ecc-a43f-5d52c42c4b27)
![image](https://github.com/GitbookIO/community/assets/39222378/9994c623-d4cd-499e-93a7-c3d1efc0eb6b)

**Comment by aikoii** [comment]
Hi @BlackWhite2000, thanks for your report! We're looking into this 🙏  and will update you here.

**Comment by sarahf16** [comment]
Just to add I am also having this issue and have one I urgently need to update. It has also happened when trying to save a custom logo. I have sent an email to support.

**Comment by taranvohra** [comment]
Hey 👋, a fix has been deployed and it should be working now. Thanks for pointing it out!
